addons/crm/models/crm_lead.py

```diff
@@ -1077,6 +1077,8 @@ def action_set_won(self):
         return True
 
     def action_set_automated_probability(self):
+        """ Update the automated probability and align probability to that value """
+        self._compute_probabilities()
```

Comment:
Just to be sure, we could also add an `ensure_one` on this action as it's targeted to be done for a single record anyway